### DISCUSSION TITLE
PDF出力

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1832] [Q&A] [answered]
このデザインを作成してrakePDFを使用することは可能ですか？

スクリーンショット画像をご覧ください

![image](https://user-images.githubusercontent.com/49851974/178637635-4c162e47-d1cb-4b07-b993-bf4d1af8626c.png)

**Comment by kmuto** [comment, marked as answer]
「できるか？」に対しての回答は「できる」です。rake pdfの場合のDTPはLaTeXを使いますが、TeXの表現範囲において困難な箇所は見当たりません。
しかし、「どう作ったらよいか？」についてはRe:VIEWとはほぼ無関係のこととなり、LaTeXでどう表現を細かく作っていくかという段階になるので、このRe:VIEWのDiscussionで続けるのは適切な場所ではありません。
「誰か作ってくれるか？」という場合、有償でできるところの紹介はできます。

**Reply by jasondee1992** [reply]
つまり、re：viewを使用してそのデザインを作成し、PDFとしてエクスポートすることは可能ですか？

**Reply by kmuto** [reply]
用途を推測しますと、HTML→PDF (書籍の形態) を求めているのであれば、Vivliostyle https://vivliostyle.org/ja/ のほうが向いているのではないでしょうか。

Re:VIEWでもreファイル→PDF(LaTeXを使う)および、reファイル→PDF(Vivliostyleを使う) の2つの方法がありますが、もともとの原稿ファイルがHTMLファイルなのであれば、直接Vivliostyleを使ったほうがスムーズです。
